content/en/developers/ide_integrations/idea/_index.md

```diff
@@ -0,0 +1,167 @@
+
+{{< img src="/developers/ide_integrations/idea/top-list.png" alt="The Top-list view" style="width:100%;" >}}
+
+Double-clicking an item in the list, or selecting **Jump to Source** item from the context menu, opens a source code editor showing where the method is defined.
+
+{{< img src="/developers/ide_integrations/idea/jump-to-source.png" alt="Select Jump to Source from the context menu" style="width:100%;" >}}
+
+To see a flame graph visualization of a method, select the **Search in Flame Graph** item from the context menu.
+
+### Call tree
+
+The call tree to the right of the method list shows the paths that lead to (and from) the selected method.
+
+{{< img src="/developers/ide_integrations/idea/call-tree.png" alt="The method call tree" style="width:100%;" >}}
+
+The default **Caller Hierarchy** view shows the callers (or predecessors) of the target method and the frequency with which they appear in the call stack.
+
+To view the callees (or successors), click **Callee Hierarchy**.
+
+{{< img src="/developers/ide_integrations/idea/callee-hierarchy.png" alt="The callee hierarchy view" style="width:100%;" >}}
+
+Right-click on a method in the call tree to see options to navigate to the source editor or flame graph.
+
+## Flame graph
+
+A flame graph is a visualization of profiling samples that shows stack traces and their relative frequency during the sample period. The Datadog plugin aggregates data collected over the requested time frame and multiple individual profiles are aggregated together. Each individual profile covers a 60 second interval within the requested time frame.
+
+{{< img src="/developers/ide_integrations/idea/flamegraph.png" alt="A flame graph showing CPU Time over the last 4 hours" style="width:100%;" >}}
+
+Each time you change the profile type, the time frame, or the environment, the Datadog plugin generates a new flame graph.
+
+You can navigate the flame graph in several ways:
+- Double-click on any frame to focus on that method and all the methods that it has called during the sampling period.
+- Use the minimap to pan around the graph.
+- Right-click on a method and select **Jump to Source** to go to the corresponding point in the source code.
+
+Hovering over a method displays a tooltip with the following information:
+- The class name and method signature
+- The package name
+- The profiling metric value and percentage breakdown.
+
+Profiling samples include stack trace and line number information. Use the **Separate Flame Graph by** button to switch between separating frames by method or line number.
+
+{{< img src="/developers/ide_integrations/idea/separate-flamegraph-by.png" alt="Use the tooltip button to separate frames by method or line number" style="width:40%;" >}}
+
+## Items of interest
```

Comment:
Perhaps change this to `Datadog Insights`?  It is the term we use in the popup list.